### ISSUE TITLE
Set option fullscreen or maximized to true will freeze contour

### ISSUE THREAD

**Issue by rhcher** [contour-terminal/contour#1155]
### Contour Terminal version

Contour Terminal Emulator 0.3.13-master-59df9d6b

### Installer source

Github: source code cloned

### Operating System

Windows 11 wsl2 Arch

### Architecture

x86-64

### Steps to reproduce

1. set fullscreen or maximized to true
2. open contour, and contour will be unresponsive

### Expected Behavior

Open contour correctly with option fullscreen or maximized is true

### Actual Behavior

contour will be unresponsive with option fullscreen or maximized is true

**Comment by Yaraslaut** [comment]
I assume that this is windows build, right ?

**Comment by rhcher** [comment]
> I assume that this is windows build, right ?

Yes, it build on Windows11, and open with shell command: `wsl -d Arch --cd ~`

**Comment by Yaraslaut** [comment]
This can be connected with https://github.com/contour-terminal/contour/issues/1093

**Comment by ferdinandyb** [comment]
Now that I put some time into finding this myself, before finding this issue: it still exists :D

**Comment by ferdinandyb** [comment]
Small note: the freeze also happens if you set the windows shortcut to Run: Maximized, instead of trying to set it in contour configs.
